VndbSharp/Models/Staff/StaffAliases.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace VndbSharp.Models.Staff
+{
+    public class StaffAliases
+    {
+        internal StaffAliases(UInt32 Id, String Name, String OriginalName)
```

Comment:
Where is this constructor used? The `ArrayOfArrayConverter` uses the constructor that only accepts an `JArray`